Added template to the bottom (#30)
ykdojo/active-repos-for-contributing: README.md

---

## Your Project

> > Make sure to include a small but informative description on what your project is about and how we can help.

Reviewer: ```suggestion
> Make sure to include a small but informative description on what this project is about.
```

> Built on: Libraries and languages your project is build with.

Reviewer: ```suggestion
Built on: Libraries and languages this project is built with.
```

> `Your GitHub repository url`

Reviewer: ```suggestion
`GitHub repository url`
```
